backend/pom.xml

```diff
@@ -208,6 +208,7 @@
         <!-- <module>extensionsupport</module> -->
 
         <module>de.metas.frontend-testing</module>
+        <module>de.metas.qualitymgmt</module>
```

Comment:
pls also add it to `backend/metasfresh-dist/dist/pom.xml`
e.g.
```xml
        <dependency>
            <groupId>.......</groupId>
            <artifactId>de.metas.qualitymgmt</artifactId>
            <version>${metasfresh.version}</version>
            <classifier>migration-postgres-sql</classifier>
            <type>zip</type>
        </dependency>
```

Reply:
@teosarca done

Reply:
@teosarca (cc @riadmen ) this only works, if at least one migrationscript exists, so we either add 1 dummy file or drop this. 
Build works with dummy file, see test here https://github.com/metasfresh/metasfresh/actions/runs/15346837505

Reply:
@riadmen  @mrie04 yes, just add 0010_placeholder.sql